<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="tests.LoginTests" tests="6" skipped="0" failures="2" errors="0" timestamp="2023-04-18T17:01:14" hostname="DESKTOP-SND699A" time="19.041">
  <properties/>
  <testcase name="loginPositive[0](User(email=vasiatal@gmail.com, password=Vasia$1234))" classname="tests.LoginTests" time="2.021"/>
  <testcase name="loginPositive[1](User(email=vasiatal@gmail.com, password=Vasia$1234))" classname="tests.LoginTests" time="1.114"/>
  <testcase name="loginPositive[2](User(email=vasiatal@gmail.com, password=Vasia$1234))" classname="tests.LoginTests" time="1.104"/>
  <testcase name="loginPositiveConfig" classname="tests.LoginTests" time="1.112"/>
  <testcase name="loginWrongEmail" classname="tests.LoginTests" time="1.375">
    <failure message="org.openqa.selenium.UnhandledAlertException: Dismissed user prompt dialog: Wrong email or password: &#10;Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'&#10;System info: host: 'DESKTOP-SND699A', ip: '192.168.1.60', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.16.1'&#10;Driver info: org.openqa.selenium.firefox.FirefoxDriver&#10;Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 111.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:buildID: 20230321111920, moz:geckodriverVersion: 0.32.0, moz:headless: false, moz:platformVersion: 10.0, moz:processID: 14036, moz:profile: C:\Users\Asus\AppData\Local..., moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}&#10;Session ID: 53fad7d8-102c-4d51-b206-8754095744f7" type="org.openqa.selenium.UnhandledAlertException">org.openqa.selenium.UnhandledAlertException: Dismissed user prompt dialog: Wrong email or password: 
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-SND699A', ip: '192.168.1.60', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.16.1'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 111.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:buildID: 20230321111920, moz:geckodriverVersion: 0.32.0, moz:headless: false, moz:platformVersion: 10.0, moz:processID: 14036, moz:profile: C:\Users\Asus\AppData\Local..., moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 53fad7d8-102c-4d51-b206-8754095744f7
	at app//manager.NGListener.onTestFailure(NGListener.java:28)
	at app//org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:96)
	at app//org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
	at app//org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:656)
	at app//org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at app//org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at app//org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at app//org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at app//org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at app//org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base@11.0.16.1/java.util.ArrayList.forEach(ArrayList.java:1541)
	at app//org.testng.TestRunner.privateRun(TestRunner.java:794)
	at app//org.testng.TestRunner.run(TestRunner.java:596)
	at app//org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at app//org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at app//org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at app//org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at app//org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at app//org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at app//org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at app//org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at app//org.testng.TestNG.runSuites(TestNG.java:1063)
	at app//org.testng.TestNG.run(TestNG.java:1031)
	at org.gradle.api.internal.tasks.testing.testng.TestNGTestClassProcessor.runTests(TestNGTestClassProcessor.java:146)
	at org.gradle.api.internal.tasks.testing.testng.TestNGTestClassProcessor.stop(TestNGTestClassProcessor.java:91)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.stop(SuiteTestClassProcessor.java:61)
	at java.base@11.0.16.1/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base@11.0.16.1/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base@11.0.16.1/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base@11.0.16.1/java.lang.reflect.Method.invoke(Method.java:566)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at com.sun.proxy.$Proxy2.stop(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker$3.run(TestWorker.java:193)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.executeAndMaintainThreadName(TestWorker.java:129)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:100)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:60)
	at org.gradle.process.internal.worker.child.ActionExecutionWorker.execute(ActionExecutionWorker.java:56)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:133)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:71)
	at app//worker.org.gradle.process.internal.worker.GradleWorkerMain.run(GradleWorkerMain.java:69)
	at app//worker.org.gradle.process.internal.worker.GradleWorkerMain.main(GradleWorkerMain.java:74)
</failure>
  </testcase>
  <testcase name="loginWrongPassword" classname="tests.LoginTests" time="1.266">
    <failure message="org.openqa.selenium.UnhandledAlertException: Dismissed user prompt dialog: Wrong email or password: &#10;Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'&#10;System info: host: 'DESKTOP-SND699A', ip: '192.168.1.60', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.16.1'&#10;Driver info: org.openqa.selenium.firefox.FirefoxDriver&#10;Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 111.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:buildID: 20230321111920, moz:geckodriverVersion: 0.32.0, moz:headless: false, moz:platformVersion: 10.0, moz:processID: 14036, moz:profile: C:\Users\Asus\AppData\Local..., moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}&#10;Session ID: 53fad7d8-102c-4d51-b206-8754095744f7" type="org.openqa.selenium.UnhandledAlertException">org.openqa.selenium.UnhandledAlertException: Dismissed user prompt dialog: Wrong email or password: 
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-SND699A', ip: '192.168.1.60', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.16.1'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 111.0.1, javascriptEnabled: true, moz:accessibilityChecks: false, moz:buildID: 20230321111920, moz:geckodriverVersion: 0.32.0, moz:headless: false, moz:platformVersion: 10.0, moz:processID: 14036, moz:profile: C:\Users\Asus\AppData\Local..., moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, moz:windowless: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 53fad7d8-102c-4d51-b206-8754095744f7
	at app//manager.NGListener.onTestFailure(NGListener.java:28)
	at app//org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:96)
	at app//org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
	at app//org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:656)
	at app//org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
	at app//org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at app//org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
	at app//org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
	at app//org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at app//org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base@11.0.16.1/java.util.ArrayList.forEach(ArrayList.java:1541)
	at app//org.testng.TestRunner.privateRun(TestRunner.java:794)
	at app//org.testng.TestRunner.run(TestRunner.java:596)
	at app//org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
	at app//org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
	at app//org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
	at app//org.testng.SuiteRunner.run(SuiteRunner.java:276)
	at app//org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at app//org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
	at app//org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
	at app//org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
	at app//org.testng.TestNG.runSuites(TestNG.java:1063)
	at app//org.testng.TestNG.run(TestNG.java:1031)
	at org.gradle.api.internal.tasks.testing.testng.TestNGTestClassProcessor.runTests(TestNGTestClassProcessor.java:146)
	at org.gradle.api.internal.tasks.testing.testng.TestNGTestClassProcessor.stop(TestNGTestClassProcessor.java:91)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.stop(SuiteTestClassProcessor.java:61)
	at java.base@11.0.16.1/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base@11.0.16.1/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base@11.0.16.1/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base@11.0.16.1/java.lang.reflect.Method.invoke(Method.java:566)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at com.sun.proxy.$Proxy2.stop(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker$3.run(TestWorker.java:193)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.executeAndMaintainThreadName(TestWorker.java:129)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:100)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:60)
	at org.gradle.process.internal.worker.child.ActionExecutionWorker.execute(ActionExecutionWorker.java:56)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:133)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:71)
	at app//worker.org.gradle.process.internal.worker.GradleWorkerMain.run(GradleWorkerMain.java:69)
	at app//worker.org.gradle.process.internal.worker.GradleWorkerMain.main(GradleWorkerMain.java:74)
</failure>
  </testcase>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[console.warn: LoginRecipes: "Falling back to a synchronous message for: https://telranedu.web.app."
console.warn: LoginRecipes: "Falling back to a synchronous message for: https://telranedu.web.app."
console.warn: LoginRecipes: "Falling back to a synchronous message for: https://telranedu.web.app."
console.warn: LoginRecipes: "Falling back to a synchronous message for: https://telranedu.web.app."
console.warn: LoginRecipes: "Falling back to a synchronous message for: https://telranedu.web.app."
console.warn: LoginRecipes: "Falling back to a synchronous message for: https://telranedu.web.app."
]]></system-err>
</testsuite>
